index.html

```diff
@@ -0,0 +1,592 @@
+                        vision suggest that inflation, once started, is eternal. Even as our own universe settled down
+                        to a comfortable homey expansion, the rest of the cosmos will continue blowing up, spinning off
+                        other bubbles endlessly, a concept known as the multiverse.</p>
+                    <p class="content">So the future of the cosmos is perhaps bright and fecund, but do not bother
+                        asking about going
+                        any deeper into the past.</p>
+                    <p class="content">We might never know what happened before inflation, at the very beginning,
+                        because inflation
+                        erases everything that came before it. All the chaos and randomness of the primordial moment are
+                        swept away, forever out of our view.</p>
+                    <p class="content">“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian
+                        astronomer who was not
+                        part of the team, “you wind up at inflation.”</p>
+                </section>
+                <section class="width-class extra-info">
+                    <p>A version of this article appears in print on March 18, 2014, Section A, Page 1 of the New York
+                        edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. <a>Order Reprints</a>
+                        |<a>&nbsp;Today’s Paper&nbsp;</a>|<a>&nbsp;Subscribe</a></p>
+                </section>
+                <section class="width-class flex">
+                    <section class="comment-btn">
+                        <button type="submit" class="btn">READ 645 COMMENTS</button>
+                    </section>
+                    <section class="social-share">
+                        <ul class="flex">
+                            <li><i class="fa fa-facebook social-icons" aria-hidden="true"></i></li>
+                            <li><i class="fa fa-twitter social-icons" aria-hidden="true"></i></li>
+                            <li><i class="fa fa-envelope social-icons" aria-hidden="true"></i></li>
+                            <li><i class="fa fa-share social-icons" aria-hidden="true"></i></li>
+                        </ul>
+                    </section>
+                </section>
+            </section>
+        </article>
+        <section class="extra">
+            <section class="space-astronomy">
+                <a href="#">More in Space and Astronomy</a>
+                <img src="assets/images/space1.png" alt="space img1">
+                <img src="assets/images/space2.png" alt="space img2">
+                <img src="assets/images/space3.png" alt="space img3">
+                <img src="assets/images/space4.png" alt="space img4">
+                <img src="assets/images/space5.png" alt="space img5">
+                <img src="assets/images/space6.png" alt="space img6">
```

Comment:
it's not allowed to take screenshots from the original page and put them in your clone please, add HTML and CSS code instead.